bouncy-bullfrogs/bouncy/mail/views.py

```diff
@@ -0,0 +1,189 @@
+from .models import Email, User
```

Comment:
With so many imports, it would be easier to navigate them if there was separation between 3rd party libraries, built-in libraries, and your own code.

Reply:
Yes, I absolutely agree. Really sorry about this and the places that lack docstrings, I've learned my lesson while working on a production-quality project and I find that it's really important to include docstrings and clear indications for things that might seem obvious in the present but aren't that obvious sometime in the future for someone trying to get started working on the project asap. Thanks a lot for the review though!